modules/javafx.graphics/src/main/java/javafx/scene/LightBase.java

```diff
@@ -56,33 +56,112 @@
- * include:
+ * The {@code LightBase} class is the base class for all objects that represent a form of light source. All light types
+ * share the following common properties that are defined in this class:
  * <ul>
- * <li>{@code color} - the color of the light source</li>
- * <li>{@code scope} - a list of nodes the light source affects</li>
- * <li>{@code exlusionScope} - a list of nodes the light source does not affect</li>
+ *   <li>{@code color} - the color of the light source</li>
+ *   <li>{@code scope} - a list of nodes the light source affects</li>
+ *   <li>{@code exlusionScope} - a list of nodes the light source does not affect</li>
  * </ul>
  *
+ * In addition to these, each light type supports a different set of properties, summarized in the following table:
+ *
+ * <div style="display:flex; flex-direction: column; justify-content: center; align-items: center">
+ * <table border="1">
+ *   <caption>Summary of Light types by attributes</caption>
+ *   <tr>
+ *     <th>Light type</th>
+ *     <th>Rotation and Direction</th>
+ *     <th>Location and Attenuation</th>
+ *   </tr>
+ *   <tr>
+ *     <td>{@link AmbientLight}</td>
+ *     <td style="text-align:center">&#10007;</td>
+ *     <td style="text-align:center">&#10007;</td>
+ *   </tr>
+ *   <tr>
+ *     <td>{@link DirectionalLight}</td>
+ *     <td style="text-align:center">&#10003;</td>
+ *     <td style="text-align:center">&#10007;</td>
+ *   </tr>
+ *   <tr>
+ *     <td>{@link PointLight}</td>
+ *     <td style="text-align:center">&#10007;</td>
+ *     <td style="text-align:center">&#10003;</td>
+ *   </tr>
+ *   <tr>
+ *     <td>{@link SpotLight}*</td>
+ *     <td style="text-align:center">&#10003;</td>
+ *     <td style="text-align:center">&#10003;</td>
+ *   </tr>
+ * </table>
+ *
+ * * Supports spotlight attenuation factors as described in its class docs.
+ * </div>
+ * <p>
+ * An application cannot add its own light types. Extending {@code LightBase} directly may lead to an
+ * {@code UnsupportedOperationException} being thrown.
+ * <p>
+ * All light types are not affected by scaling and shearing transforms.
+ *
+ * <h2><a id="Color">Color</a></h2>
+ * A light source produces a light of a single color. The appearance of an object illuminated by the light is a result
+ * of the color of the light and the material the object is made of. For example, for a blue light, a white object will
+ * appear blue, a black object will appear black, and a (255, 0, 255)-colored object will appear blue, as the light
+ * "selects" only the blue component of the material color. The mathematical definition of the material-light
+ * interaction is available in {@link javafx.scene.paint.PhongMaterial PhongMaterial}.
+ * <p>
+ * Notes about the current implementation:
+ * <ol>
+ *   <li> A black colored light is ignored since its contribution is 0.
+ *   <li> The transparency (alpha) component of a light is ignored.
+ * </ol>
+ * There are no guarantee that these behaviors will not change.
```

Comment:
This is a new specification of behavior rather than a doc reorganization or clarification of an existing note. As such, I think it should be done separately and with a CSR.

Reply:
Even if this just clarifies the current behavior and doesn't make a guarantee?

Reply:
We will sometimes clarify existing behavior in JavaFX without a CSR. If that's what you intend with this, then I might suggest moving it to an `@implNote`. Also, I prefer something more like the following for the last sentence:

```
 * These behaviors are not specified and could change in the future.
```